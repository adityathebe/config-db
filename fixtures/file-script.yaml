file:
  - type: Config
    id: $.id
    name: $.name
    transform:
      script:
        javascript: |+
          for (var i = 0; i < config.length; i++) {
            config[i].added ="a"
          }
          JSON.stringify(config)
    paths:
      - fixtures/multiple-configs.json