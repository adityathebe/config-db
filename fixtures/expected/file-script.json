[
  {
    "last_modified": "0001-01-01T00:00:00Z",
    "type": "Config",
    "name": "Config1",
    "id": "1",
    "config": {
      "added": "a",
      "id": 1,
      "name": "Config1",
      "password": "p1",
      "secret": "secret_1"
    }
  },
  {
    "last_modified": "0001-01-01T00:00:00Z",
    "type": "Config",
    "name": "Config2",
    "id": "2",
    "config": {
      "added": "a",
      "id": 2,
      "name": "Config2",
      "password": "p2",
      "secret": "secret_2"
    }
  }
]
